### PR TITLE
E2e/bulk delete

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/helpers/keys.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
+import { createClient } from 'redis';
```
Comment (ofersteinberg) [inline tests/e2e/helpers/keys.ts:1]: Better to use node-redis client (https://github.com/redis/node-redis)

File: tests/e2e/package.json [hunk 2]
```diff
@@ -37,6 +37,8 @@
     "edit-json-file": "1.7.0",
     "eslint": "7.32.0",
     "eslint-plugin-import": "2.24.2",
+    "redis": "^3.0.2",
+    "redis-rejson": "^1.0.0",
```
Comment (ofersteinberg) [inline tests/e2e/package.json:41]: With node-redis, you have bundled all modules clients

File: tests/e2e/tests/critical-path/browser/bulk-delete.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneRedisearch, ossStandaloneRedisearch.databaseName);
+        await addHashKeyApi(hashKeyParameters, ossStandaloneRedisearch);
+        await addSetKeyApi(setKeyParameters, ossStandaloneRedisearch);
+    })
+    .afterEach(async() => {
+        // Clear and delete database
+        await deleteAllKeysFromDB(dbParameters.host, dbParameters.port);
+        await deleteStandaloneDatabaseApi(ossStandaloneRedisearch);
+    });
+test('Verify that user can access the bulk actions screen by clicking on the “Bulk Actions” button in the Browser', async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/browser/bulk-delete.e2e.ts:33]: Please set a shorter name

Comment (AlenaNaboka) [reply tests/e2e/tests/critical-path/browser/bulk-delete.e2e.ts:33]: Hi Ofer, 
we usually match manual checklist with e2e tests (they have the same names). It allows us to find them in code easily and track in test documentation easily. 
We try creating test case names as short as possible. And when we change them in e2e code, need to change in whole test documentation. 
Specifically this test case name is one-line. Can we leave it as is please?

File: tests/e2e/package.json (hunk 2 above)
Comment (vlad-dargel) [reply tests/e2e/package.json]: yes, I see, thanks
deleted redis-rejson

File: tests/e2e/tests/critical-path/browser/bulk-delete.e2e.ts (hunk 3 above)
Comment (vlad-dargel) [reply tests/e2e/tests/critical-path/browser/bulk-delete.e2e.ts]: I made it a bit shorter

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**